.azure/gpu-tests-fabric.yml

```diff
@@ -59,17 +59,13 @@ jobs:
     strategy:
       matrix:
         "Fabric | latest":
-          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.10-torch2.0-cuda11.8.0"
+          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.11-torch2.1-cuda12.1.0"
           IS_NIGHTLY: "false"
```

Comment:
You can get rid of this variable now and all its usages

Reply:
why?

Reply:
We could get rid of it and forget it for a while, or we could already add the 2.2 nightly job. I don't mind either way, let me know your opinions.

Reply:
I think removing is the simple option for now. git history will always give us what was here if when we need to add it back